README.md

```diff
@@ -123,6 +123,8 @@ To build the tests, you will require the following:
 
 * [CMake](http://cmake.org), version 3.1.3 (3.2.3 for AppleClang) or later to be installed and in your PATH.
 
+NOTE: To run tests you need at least 3.8
+
```

Comment:
Now that DGSL_TEST is off by default, replace the cmake configuration command below with
`cmake -G "Visual Studio 15 2017" c:\GSL -DGSL_TEST=ON`